app/src/main/java/fr/lille1/univ/android/architecture/btrshop/scanner/domain/dialog/FormatSelectorDialogFragment.java

```diff
@@ -0,0 +1,96 @@
+package fr.lille1.univ.android.architecture.btrshop.scanner.domain.dialog;
+
+import android.app.AlertDialog;
+import android.app.Dialog;
+import android.content.DialogInterface;
+import android.os.Bundle;
+import android.support.v4.app.DialogFragment;
+
+import com.google.zxing.BarcodeFormat;
+
+import java.util.ArrayList;
+
+import fr.lille1.univ.android.architecture.btrshop.R;
+import me.dm7.barcodescanner.zxing.ZXingScannerView;
+
+public class FormatSelectorDialogFragment extends DialogFragment {
```

Comment:
Not sure about this either. Just dont display those choices and select all by default ?